mne_bids/mne_bids.py

```diff
@@ -252,7 +288,7 @@ def _participants_tsv(raw, subject_id, group, fname, verbose):
         data.update({'age': [subject_age], 'sex': [sex], 'group': [group]})
 
     # append the participant data to the existing file if it exists
```

Comment:
minor comment: I'd add to the comment something along the lines "else if write_mode=='clear', skip appending and write a new file

this would probably increase code reading speed.